Add KoP grafana dashboard (#1360)
streamnative/kop: grafana/README.md

# Grafana Dashboard

You can find the Grafana dashboard in this directory that visualizes the KoP metrics.

## Prerequisites

The dashboard has been tested with the following software versions:

* KoP >= 2.9.0
* Grafana >= v8.4.3
* Prometheus >= v2.0.0

A Prometheus data source needs to be [added](https://prometheus.io/docs/visualization/grafana/#using) before installing the dashboard.

## Installing the Dashboard

> In the Grafana UI complete the following steps:

Reviewer: ```suggestion
To install the Grafana Dashboard, follow these steps in the Grafana UI:
```
One question: which word is more accurate, GUI, console, or UI?

Author: Grafana provided a Web interface. Should we call it UI?

> 1. Use the *New Dashboard* button and click *Import*.

Reviewer: ```suggestion
1. Click **New Dashboard** and then click **Import**.
```

> 2. Upload `dashboard.json` or copy and paste the contents of the file in the textbox and click *Load*.

Reviewer: ```suggestion
2. Upload the `dashboard.json` JSON file or copy and paste the contents of the file in the textbox and then click **Load**.
```
One question, where can we add a link to the `dashboard.json` JSON file?

> 3. Set the Prometheus data source and click *Import*.

Reviewer: ```suggestion
3. Set the Prometheus data source and click **Import**.
```
